### PR TITLE
feat(linting): biome intro

### PR DESCRIPTION
**Pull request by matthew2564**
## Ticket title

_One line description_
[CB2-XXXX](https://dvsa.atlassian.net/browse/CB2-XXXX)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Delete branch after merge

### CODE REVIEW

**Review by tomcrawleyy — DISMISSED** [review]

File: src/app/api/reference-data/model/emptyObject.ts [hunk 1]
```diff
@@ -10,5 +10,4 @@
  * Do not edit the class manually.
  */
 
-export interface EmptyObject { 
-}
```
Comment (BrandonT95) [inline src/app/api/reference-data/model/emptyObject.ts]: Is this change needed?

File: src/app/api/reference-data/model/deleteItem.ts [hunk 2]
```diff
@@ -10,6 +10,6 @@
  * Do not edit the class manually.
  */
 
-export interface DeleteItem { 
-    success?: boolean;
-}
```
Comment (BrandonT95) [inline src/app/api/reference-data/model/deleteItem.ts]: Is this change needed?

File: src/app/api/reference-data/model/emptyObject.ts (hunk 1 above)
Comment (matthew2564) [reply src/app/api/reference-data/model/emptyObject.ts]: This was automatically changed by Biome. I'll roll this back and will add the API folder to the exclusion list of the linter to stop this happening again.

**Review by tomcrawleyy — APPROVED** [review]

**Review by BrandonT95 — APPROVED** [review]

**Review by pbardy2000 — APPROVED** [review]
